api/internal/utils/utils.go

```diff
@@ -206,3 +211,14 @@ func ValueEqual(a interface{}, b interface{}) bool {
 	}
 	return bytes.Equal(aBytes, bBytes)
 }
+
+// set the hash and version
+func SetHashAndVersion(hash, version string) {
```

Comment:
Can't understand why this function is required.

If you want to assign these two values from the go linker, just referencing them in the `-X` option.

Reply:
Yes. We can.